web_widget_listview_url/static/src/js/web_list_url_widget.js

```diff
@@ -0,0 +1,21 @@
+//-*- coding: utf-8 -*-
+//Copyright 2017 Therp BV <http://therp.nl>
+//License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+openerp.web_widget_listview_url = function(instance) {
+    instance.web.list.Url = instance.web.list.Column.extend({
+        PROTOCOL_REGEX: /^(?!\w+:?\/\/)/,
+        _format: function(row_data, options) {
+            var value = row_data[this.id].value;
+            if (value) {
+                return _.template(
```

Comment:
Why not use string templates here?
https://github.com/lukehoban/es6features#template-strings